scripts/release/steps/update-changelog.js

```diff
@@ -30,6 +30,29 @@ function writeChangelog({ version, previousVersion, releaseNotes }) {
   fs.writeFileSync("CHANGELOG.md", newEntry + "\n\n" + changelog);
 }
 
+function replaceVersionInBlogPost({ blogPost, version, previousVersion }) {
+  const blogPostData = fs.readFileSync(blogPost, "utf-8");
+  const newBlogPostData = blogPostData
+    .split("\n")
+    .map((line) => {
+      const trimmed = line.trim();
+      const isComments =
+        trimmed.startsWith("// ") ||
+        trimmed.startsWith("# ") ||
+        (trimmed.startsWith("<!-- ") && trimmed.endsWith(" -->")) ||
+        (trimmed.startsWith("/* ") && trimmed.endsWith(" */")) ||
+        (trimmed.startsWith("{{!-- ") && trimmed.endsWith(" --}}"));
+      if (!isComments) {
+        return line;
+      }
+      return line
+        .replace(/Prettier stable/g, `Prettier ${previousVersion}`)
```

Comment:
You use uppercase `Stable` in

https://github.com/prettier/prettier/blob/master/changelog_unreleased/javascript/pr-6319.md,

so maybe `/Prettier [Ss]table/g` or `/Prettier stable/ig`